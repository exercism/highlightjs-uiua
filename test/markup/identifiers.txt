Name Age__1 X y
