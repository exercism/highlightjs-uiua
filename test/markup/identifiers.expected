<span class="hljs-title function_ invoke__">Name</span> <span class="hljs-title function_ invoke__">Age__1</span> <span class="hljs-title function_ invoke__">X</span> <span class="hljs-title function_ invoke__">y</span>
